gandi-config/bootstrap.d/03-resize_main_disk

```diff
@@ -47,17 +48,28 @@ if which sgdisk >/dev/null && [ -b "$disk" ]; then
         if [ "$disk_guid" = '00000000-0000-0000-0000-000000000000' ]; then
             if ! sgdisk --randomize-guids "$disk" 2>&1 | logger -t "$SYSLOG_TARGET"; then
                 err_code="$?"
-                loginfo "Error randomizing GPT UIDs on disk ${disk}, return: ${err_code}"
+                msg="Error randomizing GPT UIDs on disk ${disk}, return: ${err_code}"
+                echo "$msg" | logger -t "$SYSLOG_TARGET"
                 exit "$err_code"
             fi
         fi
         # Inform kernel of partition changes.
         if ! partx --update "$disk" 2>&1 | logger -t "$SYSLOG_TARGET"; then
             err_code="$?"
-            loginfo "Error updating in-kernel partition layout for disk " \
+            msg="Error updating in-kernel partition layout for disk " \
                 "${disk}, return: ${err_code}"
+            echo "$msg" | logger -t "$SYSLOG_TARGET"
             exit "$err_code"
         fi
+        # partx is sometimes not enough, e.g. in ubuntu 20 it does not trigger
+        # /dev/disk/by-partlabel links creation
+        if ! partprobe "$disk" 2>&1 | logger -t "$SYSLOG_TARGET"; then
```

Comment:
you should add dependency to partprobe in package or check for partprobe presence

Reply:
Added a check on command partprobe